### ISSUE TITLE
Chinese handicap : "waiting for you to play all your handicap stones" message

### ISSUE THREAD

**Issue by wonderingabout** [online-go/gtp2ogs#80]
@roy7 @ anoek 
i noticed several players dont realize they have to play all the handicap stones when the game starts

after they play one stone, they wait a long time, and sometimes they even leave the game

it would be good to add a bot message during the game telling the player that more stones need to be added

for `n` handicap stones, at every `x` number of handicap stone played

```
"you put x handicap stones, please put n-x more handicap stones"
```
example for a 6 handicap stones game: 

```
"you put 1 handicap stones, please put 5 more handicap stones"
```

something based on if and else :)

my most recent example, this player https://online-go.com/game/15885917 waited like 3 minutes before playing his 2nd handicap stone

**Comment by roy7** [comment]
The bot won't even start though until it is the bot's move. OGS indicates to the other player it is their turn. It'll show they have a game waiting in top right corner. I don't think this is a gtp2ogs issue.

**Comment by wonderingabout** [comment]
i noticed this behaviour in several games though

the problem is since the clock doesnt start and nothing moves, the human player sometimes assume the bot is thinking long for its first move, or the bot has crashed and leaves

my first message may have been complicated, but i thought of something easier : 

if 
rules = chinese and notification.handicap > 0
write bot greeting message
"hello, this is a handicap game, please put your notification.handicap stones on the board"

see screenshots below

![handi1](https://user-images.githubusercontent.com/38690718/50508166-fbe9b580-0a80-11e9-94d1-b8ec184cb719.png)
![handi2](https://user-images.githubusercontent.com/38690718/50508165-fbe9b580-0a80-11e9-83ff-0c5c703b254a.png)
![handi3](https://user-images.githubusercontent.com/38690718/50508164-fbe9b580-0a80-11e9-8bf2-ce46ff43414c.png)
![handi4](https://user-images.githubusercontent.com/38690718/50508163-fbe9b580-0a80-11e9-840b-6949b76e5a8f.png)

it definitely can be confusing from a human player perspective

**Comment by roy7** [comment]
Ahh ok I thought clock started when you placed the first stone, my mistake.

**Comment by wonderingabout** [comment]
its ok :) 
(mail notifications with thunderbird are sure helpful)

**Comment by wonderingabout** [comment]
happened again today...

i'll try to implement this on gtp2ogs

**Comment by wonderingabout** [comment]
fixed in #118 

example 

![works great](https://user-images.githubusercontent.com/38690718/52262742-9e714000-292d-11e9-863c-23a55691ddbc.png)
